src/NHSD.BuyingCatalogue.Solutions.API/Controllers/PriceController.cs

```diff
@@ -49,33 +48,39 @@ public async Task<ActionResult<PricingResult>> GetListAsync(string solutionId)
             {
                 Id = solutionId,
                 Name = prices.First()?.CatalogueItemName,
-                Prices = prices.Select(x => new PriceResult
-                {
-                    PriceId = x.CataloguePriceId,
-                    Type = x.Type,
-                    CurrencyCode = x.CurrencyCode,
-                    ItemUnit = new ItemUnitResult
-                    {
-                        Name = x.PricingUnit.Name,
-                        Description = x.PricingUnit.Description,
-                        TierName = x.PricingUnit.TierName
-                    },
-                    TimeUnit = x.TimeUnit is null ? null : new TimeUnitResult
-                    {
-                        Name = x.TimeUnit.Name,
-                        Description = x.TimeUnit.Description
-                    },
-                    Price = (x as FlatCataloguePriceDto)?.Price,
-                    Tiers = (x as TieredCataloguePriceDto)?.TieredPrices.Select(x => new TierResult
-                    {
-                        Start = x.BandStart,
-                        End = x.BandEnd,
-                        Price = x.Price
-                    })
-                })
+                Prices = prices.Select(GetPriceResult)
```

Comment:
can you add a new line after the })

Reply:
Not sure what you mean here there is already  new line after the closing bracket? Do you mean remove the new line ?

`public async Task<ActionResult<PricingResult>> GetListAsync(string solutionId)
        {
            var prices = (await _mediator.Send(new GetPriceBySolutionIdQuery(solutionId))).ToList();

            var result = new PricingResult
            {
                Id = solutionId, Name = prices.First()?.CatalogueItemName, Prices = prices.Select(GetPriceResult)
            };

            return result;
        }`